Cargo.toml

```diff
@@ -1,59 +1,3 @@
-[package]
-name = "opentelemetry"
-version = "0.9.1"
-authors = ["OpenTelemetry Authors <cncf-opentelemetry-contributors@lists.cncf.io>"]
-description = "A metrics collection and distributed tracing framework"
-homepage = "https://github.com/open-telemetry/opentelemetry-rust"
-repository = "https://github.com/open-telemetry/opentelemetry-rust"
-readme = "README.md"
-categories = [
-    "development-tools::debugging",
-    "development-tools::profiling",
-    "api-bindings",
-    "asynchronous",
-]
-keywords = ["opentelemetry", "logging", "tracing", "metrics", "async"]
-license = "Apache-2.0"
-edition = "2018"
-
-[package.metadata.docs.rs]
-all-features = true
-rustdoc-args = ["--cfg", "docsrs"]
-
-[dependencies]
-async-std = { version = "1.6", features = ["unstable"], default-features = false, optional = true  }
-async-trait = { version = "0.1", optional = true }
-base64 = { version = "0.13", optional = true }
-bincode = { version = "1.2", optional = true }
-dashmap = { version = "4.0.0-rc6", optional = true }
-fnv = { version = "1.0", optional = true }
-futures = "0.3"
-lazy_static = "1.4"
-percent-encoding = { version = "2.0", optional = true }
-pin-project = { version = "0.4", optional = true }
-rand = { version = "0.7", default-features = false, features = ["std"], optional = true }
-regex = { version = "1.3", default-features = false, features = ["std", "perf"], optional = true}
-serde = { version = "1.0", features = ["derive", "rc"], optional = true }
-http = { version = "0.2", optional = true }
-thiserror = { version = "1.0", optional = true }
-tokio = { version = "0.2", default-features = false, features = ["rt-core", "blocking", "time", "stream"], optional = true }
-tonic = { version = "0.3", default-features = false, optional = true }
-reqwest = { version = "0.10", default-features = false, features = ["blocking"], optional = true }
-surf = { version = "2.0", default-features = false, optional = true }
-
-[dev-dependencies]
-criterion = "0.3.1"
-rand_distr = "0.3.0"
-tokio = { version = "0.2", features = ["full"] }
-
-[features]
-default = ["trace"]
-base64_format = ["base64", "binary_propagator"]
-trace = ["rand", "pin-project", "async-trait", "regex", "percent-encoding"]
-metrics = ["thiserror", "dashmap", "fnv"]
-serialize = ["serde", "bincode"]
-binary_propagator = []
-
 [workspace]
 members = [
```

Comment:
Do we need to add the new `opentelemetry` folder to the workspace members?

Reply:
Huh, good catch!